package.json

```diff
@@ -116,7 +116,8 @@
     "lodash.template": "^4.5.0",
     "websocket-extensions": "^0.1.4",
     "bl": "^3.0.1",
-    "dot-prop": "^4.2.1"
+    "dot-prop": "^4.2.1",
+    "ini": "^1.3.7"
```

Comment:
would this be accompanied with a yarn lock update?

Reply:
added updated yarn.lock